### ISSUE TITLE
Questions about EEGDataset class in 'dataset_EEG.py'

### ISSUE THREAD

**Issue by gyuseok0917** [LuigiSigillo/GWIT#7]
Hi.
Thank you for sharing your great work.

While analyzing your dataset code, I was looking at the EEGDataset in dataset_EEG.py. I saw that in this class, you are calculating the mean and standard deviation of the entire EEG and then normalizing it. However, as far as I know, the Spampinato et al. dataset is already normalized, so why is it normalized again?

```
class EEGDataset(Dataset):
    
    # Constructor
    def __init__(self, eeg_signals_path, image_transform=identity, subject=4, 
                 encoder_name = 'bendr', imagenet_path = '/mnt/media/luigi/dataset/imageNet_images/',
                   only_eeg=False):             
        """
        ...
        """
        
        self.data_l = [torch.tensor(item["eeg"])[:,20:460] for item in self.data]
        self.mean = torch.mean(torch.stack(self.data_l))
        self.std = torch.std(torch.stack(self.data_l))
        del self.data_l

    # Get item
    def __getitem__(self, i):
        # Process EEG
        """
        ...
        """
            
       #normalize eeg
       eeg = (eeg - self.mean) / self.std
```

**Comment by LuigiSigillo** [comment]
Hi, those are old dataset classes and are actually not used in the project. 
The dataset is loaded using the hf repo dataset on which i uploaded the files according to how controlnet handles the inputs.
I confirm we used the Spampinato et al dataset as it is provided by them
